From: Yang Yingliang <yangyingliang@huawei.com>
Subject: mm/damon: change damon_lru_sort_stub_pattern to static
Date: Sat, 17 Sep 2022 20:12:28 +0800

damon_lru_sort_stub_pattern is only used in lru_sort.c now, change it to
static.

Link: https://lkml.kernel.org/r/20220917121228.1889699-1-yangyingliang@huawei.com
Fixes: 27812ec2a990 ("mm/damon: simplify scheme create in lru_sort.c")
Signed-off-by: Yang Yingliang <yangyingliang@huawei.com>
Cc: SeongJae Park <sj@kernel.org>
Cc: Xin Hao <xhao@linux.alibaba.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 mm/damon/lru_sort.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/mm/damon/lru_sort.c~mm-damon-simplify-scheme-create-in-lru_sortc-fix
+++ a/mm/damon/lru_sort.c
@@ -132,7 +132,7 @@ DEFINE_DAMON_MODULES_DAMOS_STATS_PARAMS(
 		lru_sort_tried_cold_regions, lru_sorted_cold_regions,
 		cold_quota_exceeds);
 
-struct damos_access_pattern damon_lru_sort_stub_pattern = {
+static struct damos_access_pattern damon_lru_sort_stub_pattern = {
 	/* Find regions having PAGE_SIZE or larger size */
 	.min_sz_region = PAGE_SIZE,
 	.max_sz_region = ULONG_MAX,
_
